mule-extensions-api-persistence/src/test/resources/metadata/success-result-operation-descriptor.json

```diff
@@ -71,6 +71,11 @@
             "hasDynamicType": true,
             "required": false,
             "isConfigOverride": false,
+            "valuesProviderModel": {
+              "requiredParameters": [],
```

Comment:
this array is always empty in all tests. Should have at least one test with several